docker/README.md

```diff
@@ -2,36 +2,36 @@
 # Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04
 
 
-We provide Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04 which can be used to build runtime Docker* images for OpenVINO™ integration with TensorFlow on CPU, GPU, VPU, and HDDL.
+We provide Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04 which can be used to build runtime Docker* images for OpenVINO™ integration with TensorFlow on CPU, GPU, VPU, and VAD-M.
 They contain all required runtime python packages, and shared libraries to support execution of a TensorFlow Python app with the OpenVINO™ backend. By default, it hosts a Jupyter server with an Image Classification and an Object Detection sample that demonstrate the performance benefits of using OpenVINO™ integration with TensorFlow on the CPU.
 
 Build the docker image
 
-	$ docker build -t openvino_tensorflow/ubuntu20_runtime:2.0.0 - < ubuntu20/openvino_tensorflow_cgvh_runtime_2.0.0.dockerfile
+	$ docker build -t openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0 - < ubuntu20/openvino_tensorflow_cgvh_runtime_2.0.0.dockerfile
 
 Launch the Jupyter server with
 
-	$ docker run -it -p 8888:8888 openvino_tensorflow/ubuntu20_runtime:2.0.0
+	$ docker run -it -p 8888:8888 openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0
 
-To get iGPU,and MYRIAD support
+To get iGPU and MYRIAD support
 
-	$ docker run -it -p 8888:8888 -v /dev:/dev --network=host --privileged openvino_tensorflow/ubuntu20_runtime:2.0.0
+	$ docker run -it -p 8888:8888 -v /dev:/dev --network=host --device /dev/dri:/dev/dri --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0
 
```

Comment:
Can we have separate commands for iGPU and Myriad since they need different device accesses? For example, if the developer does not have an iGPU, they don't have to provide access to /dev/dri. How about VAD-M? Will the same command work for VAD-M? I believe we may need to provide different accesses for VAD-M.

Reply:
I don't see environment variables for ONEDNN and CONVERT_VARIABLES_TO_CONSTANTS enabled in Dockerfile. Are they enabled by default somewhere? Also how about the prebuilt Docker images? Since these are not enabled by default in the source, does it make sense for us to include them to get the best performance in Dockerfiles and images?

Reply:
> I don't see environment variables for ONEDNN and CONVERT_VARIABLES_TO_CONSTANTS enabled in Dockerfile. Are they enabled by default somewhere? Also how about the prebuilt Docker images? Since these are not enabled by default in the source, does it make sense for us to include them to get the best performance in Dockerfiles and images?

Yes, these environment variables would be added, in Dockerfiles, ipynb, as well as to all other samples.

Reply:
These environmental variables are now enabled in classification and object detection samples and notebooks.